src/Faker/English/FileExtension.php

```diff
@@ -0,0 +1,578 @@
+        'video/vnd.dece.pd'                                                         => ['uvp', 'uvvp'],
+        'video/vnd.dece.sd'                                                         => ['uvs', 'uvvs'],
+        'video/vnd.dece.video'                                                      => ['uvv', 'uvvv'],
+        'video/vnd.dvb.file'                                                        => 'dvb',
+        'video/vnd.fvt'                                                             => 'fvt',
+        'video/vnd.mpegurl'                                                         => ['mxu', 'm4u'],
+        'video/vnd.ms-playready.media.pyv'                                          => 'pyv',
+        'video/vnd.uvvu.mp4'                                                        => ['uvu', 'uvvu'],
+        'video/vnd.vivo'                                                            => 'viv',
+        'video/webm'                                                                => 'webm',
+        'video/x-f4v'                                                               => 'f4v',
+        'video/x-fli'                                                               => 'fli',
+        'video/x-flv'                                                               => 'flv',
+        'video/x-m4v'                                                               => 'm4v',
+        'video/x-matroska'                                                          => ['mkv', 'mk3d', 'mks'],
+        'video/x-mng'                                                               => 'mng',
+        'video/x-ms-asf'                                                            => ['asf', 'asx'],
+        'video/x-ms-vob'                                                            => 'vob',
+        'video/x-ms-wm'                                                             => 'wm',
+        'video/x-ms-wmv'                                                            => 'wmv',
+        'video/x-ms-wmx'                                                            => 'wmx',
+        'video/x-ms-wvx'                                                            => 'wvx',
+        'video/x-msvideo'                                                           => 'avi',
+        'video/x-sgi-movie'                                                         => 'movie',
+    ];
+
+    /**
+     * Get a random MIME type
+     *
+     * @example 'video/avi'
+     */
+    public function mimeType(): string
+    {
+        return array_rand($this->mimeTypes, 1);
+    }
+
+    /**
+     * Get a random file extension (without a dot)
+     *
+     * @example avi
+     */
+    public function extension(): string
+    {
+        $extension = $this->mimeTypes[array_rand($this->mimeTypes, 1)];
+
+        return is_array($extension) ? $extension[array_rand($extension, 1)] : $extension;
+    }
+
+    /**
+     * Get a full path to a new real file on the system.
+     */
+    public function filePath(): string
```

Comment:
`file()` is removed/replaced on purpose?

Reply:
Yes. Maybe you can help me understand the current implementation of `file()`.

I saw that it was taking a random file in `/tmp` and copied it to a new location. That is both slow and not really preferred. Also, what if the operating system does not have a `/tmp`. 

I saw so many issues with it, that is why I replaced it. But I might have missed something.

Reply:
Yes, it seems to just copy a random file from one directory to another, by default `/tmp`. 

No idea whether this one is being used a lot or not, guess it's fine to just remove it and wait for somebody to complain. I'll update https://github.com/Nyholm/next-faker to reflect that change.